src/librbd/cache/pwl/AbstractWriteLog.h

```diff
@@ -63,8 +64,9 @@ class AbstractWriteLog {
 public:
   typedef io::Extent Extent;
   typedef io::Extents Extents; 
+  Builder *m_builder;
 
-  AbstractWriteLog(ImageCtxT &image_ctx, librbd::cache::pwl::ImageCacheState<ImageCtxT>* cache_state);
+  AbstractWriteLog(ImageCtxT &image_ctx, librbd::cache::pwl::ImageCacheState<ImageCtxT>* cache_state, Builder *builder);
```

Comment:
Nit: try to keep lines to at most 80 characters lone

Reply:
Done